### DISCUSSION TITLE
Find-DbaLoginInGroup:  Include DB Users with no SQL Server Login

### DISCUSSION THREAD

**Discussion by mscirri** [dataplat/dbatools#7402] [Ideas] [answered]
Is it possible to update this function to (maybe optionally) also get WindowsGroup accounts that are database users but do NOT have a SQL login for the instance?   I tried just copying the function and adding the code in myself but it references a bunch of functions that are not exported. I am going to move on to another approach but thought this might be a useful add.

**Comment by andreasjordan** [comment, marked as answer]
This would be a feature request, so please open an issue for that here: https://github.com/sqlcollaborative/dbatools/issues
Then this can be discussed with others and maybe lead to a pull request to change the code. The more code you supply the more likely the new feature gets added.

**Reply by mscirri** [reply]
Understood. I am going to try to get to that by the end of the week.  Thank you for the information.
